### DISCUSSION TITLE
lag?

### DISCUSSION THREAD

**Discussion by blackwing12344** [File-New-Project/EarTrumpet#999] [Q&A] [answered]
when I open the taskbar flyout, it opens at about 2 fps, is anyone else getting this?

**Comment by riverar** [comment]
Can you provide a video perhaps?

**Comment by blackwing12344** [comment]
https://user-images.githubusercontent.com/77736883/149021626-bd663d89-592b-4a83-a118-a340dd761bf2.mp4

**Reply by riverar** [reply]
Ah ok. Got it.

**Comment by riverar** [comment, marked as answer]
Sorry about that. This is a complicated and known issue #391 #473 that we don't currently have a good fix for yet.

**Comment by almanaccare** [comment]
I'm also experiencing this. Hoping for a fix in the near-term!
